stage #2/module #4 String/lesson #2. StringBuilder & StringBuffer/content.md

```diff
@@ -0,0 +1,164 @@
+# StringBuffer (java.lang.StringBuffer)
```

Comment:
Please describe StringBuilder first or change the name of the lesson to smth like "StringBuffer vs StringBuilder"

Reply:
adjusted, thanks